src/TVC8.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Can we replace this with the new tests I suggested?

Reply:
Moved to #90